packages/components/breadcrumbs/breadcrumbs.ts

```diff
@@ -198,41 +192,73 @@ export class KbqBreadcrumbs implements AfterContentInit {
      * When disabled, user interactions are blocked.
      */
     @Input({ transform: booleanAttribute }) disabled: boolean = false;
+    /**
+     * Wrapping behavior of the breadcrumb items.
+     */
+    @Input() wrapMode: KbqBreadcrumbsWrapMode = this.configuration.wrapMode;
 
     @ContentChild(KbqBreadcrumbsSeparator, { read: TemplateRef })
     protected readonly separator?: TemplateRef<any>;
 
     @ContentChildren(forwardRef(() => KbqBreadcrumbItem))
     protected readonly items: QueryList<KbqBreadcrumbItem>;
 
+    @ViewChild(KbqOverflowItemsResult, { read: ElementRef })
+    protected readonly result: ElementRef;
```

Comment:
используется же только в классе
```suggestion
    private readonly result: ElementRef;
```